PSR-2-R-coding-style-guide-additions.md

```diff
@@ -79,14 +79,17 @@ If you intend to use the keywords in template files, you should at least be cons
 But in general it is better to also stick to curly brackets here for consistency throughout the codebase.
 
 ## PHP Open Tags
-Always use `<?php` instead of `<?`.
+Always use `<?php` or `<?=` instead of `<?`.
 
-Avoid using the `<?= ... ?>` short tags. They are also difficult to comment out if desired.
-It is better to consistently use
-```html
-This <?php echo h($var); ?>
+The `<?=` form should be used for echoing values of simple variables while the `<?php` form can be used for more complex code.
 
-Another <?php //echo h($var); ?> commented out one
+The short form is supported by the newest PHP versions as well and it makes your files less verbose, thus easier to read. It doesn't even require the semicolon (`;`) so feel free to omit it. 
+```html
+This <?= $x ?> way.
+```
+If you want to comment it out then you can do it easily:
+```html
+This <?//= $x ?> way.
```

Comment:
Rather than printing nothing, this prints the code! (at least for sensible systems with short open tag disabled).

Reply:
You are totally right, you can make a PR to remove this part.

Reply:
Obviously, if short tags are not allowed then they should not be used. :)

But I think this decision should be let for the team and not prohibit it in this standard. Just imagine a system where zillions of variables are inserted into template files. The short opening tags make those files sooo much simpler and cleaner. 

So, I think it should be emphasized here that using short tags cause such a risk and the developer should take the responsibility if they still want to use it. 

<offtopic>
By the way, this is something that can be interesting for you: https://youtrack.jetbrains.com/issue/WI-38482 :)
</offtopic>